### DISCUSSION TITLE
Missing parameters from ZW090 AEON Labs / Z‐Stick Gen5 USB

### DISCUSSION THREAD

**Discussion by bchabrier** [zwave-js/zwave-js#3698] [Q&A]
Hello,

I am using a ZW090 AEON Labs / Z‐Stick Gen5 USB.

Looking at the Control Board in ZWaveJS2MQTT, I correctly see the controller. However, I am not able to see (and change) the configuration parameters I should normally see, according to the documentation https://devices.zwave-js.io/?jumpTo=0x0086:0x0001:0x005a:0.0

My expectation is to have access to e.g. configuration parameter 81 to disable the LED indicator.

Is there something I am missing or doing wrong?

**Comment by AlCalzone** [comment]
You cannot use configuration params to control a stick connected via USB. You can try sending a custom message via the driver function feature in zwavejs2mqtt. This example highlights your exact use case: https://zwave-js.github.io/node-zwave-js/#/usage/custom?id=send-a-custom-message-to-the-controller

You might have to add additional imports before the shown code (@robertsLando is that correct?):
```ts
const { Message, MessageType, MessagePriority} = this.require("zwave-js")
```

**Reply by AlCalzone** [reply]
The low level solution would be executing this on the console (don't forget to use the actual path):
```bash
echo -e "\x01\x08\x00\xF2\x51\x01\x00\x05\x01\x51" > /dev/ttyUSB0
```

**Reply by robertsLando** [reply]
> You might have to add additional imports before the shown code (@robertsLando is that correct?):

Yep that's correct to use it in a driver function

**Reply by bchabrier** [reply]
Thanks for the prompt answer(s)! It gives me access to the stick functionality through custom messages.

However the custom messages doc says `It may be necessary to send messages or commands that aren't implemented yet.`. Does it mean that for my stick the commands are (still) not implement and might be in the future if someone feels brave enough to improve the config DB for this device?

**Reply by AlCalzone** [reply]
This isn't a matter of configuration files. Controlling the stick happens through several standardized commands which have a defined binary representation and cause a certain thing to happen, e.g. sending some bytes via radio to a node.

Now some stick firmwares support **non-standard** commands like the one above. These differ between the different sticks, for example ZWave.me UZB sticks have their own firmware with extra commands on top of the official commands.

These aren't implemented in the driver (yet) and require you to define them manually (like in the linked examples) if you want to send them. You will also not be able to easily match potential responses to the sent commands.

There is an open issue to track support for this https://github.com/zwave-js/node-zwave-js/issues/1245, but honestly it isn't very high on my list of priorities.
